src/core/server/events/events.ts

```diff
@@ -106,7 +120,26 @@ export type CommentReleasedCoralEventPayload = CoralEventPayload<
 
 export const CommentReleasedCoralEvent = createCoralEvent<
   CommentReleasedCoralEventPayload
->(CoralEventType.COMMENT_RELEASED);
+>(CoralEventType.COMMENT_RELEASED, {
+  forward: [CommentEnteredCoralEvent],
+});
+
+export interface CommentReplyReleasedInput {
+  ancestorIDs: string[];
+  commentID: string;
+  storyID: string;
+}
```

Comment:
This type is not used outside this invocation below for the `CommentReplyReleasedCoralEventPayload`. Inline this within the `CommentReplyReleasedCoralEventPayload` as others in this file have done instead.